packages/website/docs/writing-package-docs.md

```diff
@@ -12,6 +12,24 @@ The contents of your README will also be displayed alongside this info.
 
 Any `.md` or `.mdx` file placed inside the `/docs` folder within your package will be included as pages in your package documentation. This is a good place to put tutorials and API documentation for your package.
 
+#### Frontmatter
+
+You can add a frontmatter header to your markdown to add metadata related to the document. This must be specified as valid YAML between triple-dashed lines at the very top of the file, e.g.
+
+```
+---
+title: My awesome title
+---
+
+# Markdown content here
+```
+
+Supported metadata keys are:
+
+- title - Change the default title of your document in the sidebar nav and breadcrumbs
+
+**Note**: Changes to metafiles will not take effect on the website without a restart.
```

Comment:
Docs 🎉

Question not answered here, can I access these bits of meta-info from my pages, or is it only ever used internal to brisk?

If it's only internal, what can I use? Otherwise, how can I access it?

Reply:
Yep, internal only, I've updated the docs to make the supported fields more clear.